tuf/api/metadata.py

```diff
@@ -1287,7 +1287,8 @@ def __init__(
 
     @property
     def custom(self) -> Any:
-        return self.unrecognized_fields.get("custom", None)
+        """Contains additional information about the TargetFile instance"""
```

Comment:
Hmm, let's try to provide something more actionable here for someone just reading the API: now it sounds like python-tuf provides this additional information.

Maybe something like:
```
    """Can be used to provide implementation specific data related to the
    target. python-tuf does not use or validate this data.
    """
```